webrender/src/box_shadow.rs

```diff
@@ -26,6 +27,17 @@ pub const MAX_BLUR_RADIUS : f32 = 300.;
 // blurred without being affected by the border radius.
 pub const MASK_CORNER_PADDING: f32 = 4.0;
 
+#[derive(Debug, Copy, Clone, Eq, Hash, PartialEq)]
+pub struct BoxShadowCacheKey {
+    pub blur_radius: Au,
+    pub spread_radius: Au,
+    pub offset_x: Au,
+    pub offset_y: Au,
+    pub color: PremultipliedColorF,
```

Comment:
I don't think color needs to be part of the cache key - the box shadows are rendered as an alpha mask and the color is applied when drawing the mask to the screen.

Reply:
Okay, I will remove color from the cache key.